Tools/scripts/summarize_stats.py

```diff
@@ -115,6 +116,64 @@ def save_raw_data(data: RawData, json_output: TextIO):
     json.dump(data, json_output)
 
 
+class Doc:
+    def __init__(self, text: str, doc: str):
+        self.text = text
+        self.doc = doc
```

Comment:
This and `Ratio` could possibly be simplified slightly if you made them dataclasses